src/catalogue/category/property/addPropertyMigrationDialog.component.tsx

```diff
@@ -0,0 +1,683 @@
+                        valueType: `${property.type}_${property.mandatory}`,
+                        value: {
+                          av_placement_id: defaultValue.value.av_placement_id,
+                          value: event.target.value,
+                        },
+                      });
+                    }}
+                    error={!!errors?.default_value?.value?.value}
+                    helperText={
+                      errors?.default_value?.value?.value?.message as string
+                    }
+                    fullWidth
+                  />
+                );
+              }}
+            />
+          )}
+          <Controller
+            control={control}
+            name={`unit_id`}
+            render={({ field: { value, onChange } }) => (
+              <Autocomplete
+                disabled={
+                  property.type === CatalogueCategoryPropertyType.Boolean
+                }
+                id={crypto.randomUUID()}
+                options={units ?? []}
+                getOptionLabel={(option) => option.value}
+                value={units?.find((unit) => unit.id === value) || null}
+                fullWidth
+                onChange={(_event, unit) => {
+                  onChange(unit?.id ?? null);
+                }}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    label="Select Unit"
+                    variant="outlined"
+                    disabled={
+                      property.type === CatalogueCategoryPropertyType.Boolean
+                    }
+                  />
+                )}
+              />
+            )}
+          />
+          <Controller
+            control={control}
+            name={`mandatory`}
+            render={({ field: { value, onChange } }) => (
+              <Autocomplete
+                disableClearable
+                id={crypto.randomUUID()}
+                value={value === 'true' ? 'Yes' : 'No'}
+                onChange={(_event, value) => {
+                  const newValue = value === 'Yes' ? 'true' : 'false';
+                  setValue(
+                    'default_value.valueType',
+                    `${property.type}_${newValue}`
+                  );
+                  clearErrors('default_value.value');
+                  onChange(newValue);
+                }}
+                fullWidth
+                options={['Yes', 'No']}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    label="Select is mandatory?"
+                    variant="outlined"
+                  />
+                )}
+              />
+            )}
+          />
+        </Stack>
+      </DialogContent>
+      <DialogActions>
+        <Grid container px={1.5}>
+          <Grid item sx={{ width: '100%' }}>
+            <MigrationWarningMessage
+              isChecked={isTermsAndConditionsChecked}
```

Comment:
```suggestion
              isChecked={isMigrationWarningChecked}
```
Just to be more relevant - same for set below.